payments-core/src/main/java/com/stripe/android/view/ActivityStarter.kt

```diff
@@ -72,6 +73,14 @@ abstract class ActivityStarter<TargetActivityType : Activity, ArgsType : Activit
         }
     }
 
+    // Activity Starter Args that are required to build a form controller
+    // see [FormControllerModule]
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    interface BaseFormArgs : Args {
+        val merchantName: String
+        val stripeIntent: StripeIntent
+    }
```

Comment:
I think it's better to pass the arguments (merchantName, stripeIntent) individually into `FormControllerSubcomponent` instead of inheriting from `ActivityStarter.Args`. The `FormController` is not launched in a new Activity, so it doesn't fit this hierarchy. That also would decouple more the two if `LinkActivityContract.Args` doesn't inherit `ActivityStarter.BaseFormArgs`.